config/routes.rb

```diff
@@ -62,6 +62,8 @@ def matches?(request)
       resources :school_search, only: [:create]
     end
   end
+  # Temporary start page for Maths & Physics
+  get "maths-and-physics/start", to: "static_pages#maths_and_physics", defaults: {policy: "maths-and-physics"}
```

Comment:
Do we need to set up a controller action for this?

Reply:
Technically no, because of [default rendering of controller actions](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action). We've defined the methods for the other static pages mainly to aid with visibility and awareness, but they're not technically necessary either. I skipped it for this one because it is a temporary page that will be removed in the next couple months. I can add it in though if you think it would be helpful?

Reply:
Yeah, I think that's what threw me. I'd rather we went with one approach or the other TBH